test/e2e/suite/issuers/acme/issuer.go

```diff
@@ -267,4 +273,76 @@ var _ = framework.CertManagerDescribe("ACME Issuer", func() {
 			})
 		Expect(err).NotTo(HaveOccurred())
 	})
+	It("ACME account with External Account Binding", func() {
+
+		By("providing the legacy keyAlgorithm value")
+
+		var (
+			secretName = "test-secret"
+			keyID      = "kid-1"
+			key        = "kid-secret-1"
+		)
+
+		// TODO: this value will get base64 encoded twice. Investigate why we need to do this.
```

Comment:
This is expected - the two layers of base64 encoding are because of:

1) the ACME spec recommends encoding with base64 because a key can contain arbitrary `[]byte` data (which may not render well)
2) all `[]byte` data in JSON/YAML is encoded to base64 by encoders

One option could have been for us to make the field a `string` instead of `[]byte`, which would have avoided incurring the (2) above - but in cases where a server is only _semi compliant_ and does in fact use non-b64 encoded keys, we'd then not be able to represent this well

Reply:
Thanks for the explanation! I guess what confused me was that when I was creating an Issuer by hand from yaml, I also had to base64 encode it twice- but I guess that would be because the value the users would receive from the other CA would already be base64-encoded. I have removed the TODO.